corehq/apps/registration/utils.py

```diff
@@ -194,6 +155,9 @@ def send_domain_registration_email(recipient, domain_name, guid, full_name):
         "forum_link": FORUM_LINK,
         "wiki_link": WIKI_LINK,
         'url_prefix': '' if settings.STATIC_CDN else 'http://' + DNS_name,
+        "is_mobile_experience": (
+            toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST_CONTROLLER.enabled(recipient) and
+            toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST.enabled(recipient, toggles.NAMESPACE_USER))
```

Comment:
Is providing the namespace necessary? It caught my eye that the two `enabled` calls were different.

Reply:
Yeah. It took me awhile to figure out, but the way predictably random toggles determine if a user is in the test is the [hash](https://github.com/dimagi/commcare-hq/blob/jr/saas/make-mobile-great-again/corehq/toggles.py#L204-L206) of a [few properties](https://github.com/dimagi/commcare-hq/blob/jr/saas/make-mobile-great-again/corehq/toggles.py#L240-L241), which includes the namespace.